### PR TITLE
Voter Registration Marketing Page

### PR DESCRIPTION
**Pull request by mendelB**
### What's this PR do?

This pull request adds a new Voter Registration Marketing Page (VR landing pages for marketing-partners and influencers).

### How should this be reviewed?
👀 

### Any background context you want to provide?
The pages will be created/customized via Contentful, but this gets started on the front-end with stubbed data.

### Relevant tickets

References [Pivotal #176905654](https://www.pivotaltracker.com/story/show/176905654).

### Checklist

- [x] This PR has been added to the relevant Pivotal card.
- [ ] Documentation added for new features/changed endpoints.
- [x] Added screenshots of front-end changes on small, medium, and large screens.
- [x] Added appropriate feature/unit tests.

image

- [Medium](https://user-images.githubusercontent.com/12417657/108237969-7456d900-7116-11eb-9765-6fd383d48436.png)

- [Small](https://user-images.githubusercontent.com/12417657/108238018-820c5e80-7116-11eb-8ede-300b60ee92c2.png)

### CODE REVIEW

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import SiteFooter from '../../utilities/SiteFooter/SiteFooter';
+import SocialShareTray from '../../utilities/SocialShareTray/SocialShareTray';
+import SiteNavigationContainer from '../../SiteNavigation/SiteNavigationContainer';
+import {
+  contentfulImageUrl,
+  contentfulImageSrcset,
+} from '../../../helpers/contentful';
+
+const coverImage =
+  'https://images.ctfassets.net/81iqaqpfd8fy/2bdz45WOkQRw0X0bzS6DJ6/0d3af059a19eec329c34aa2935866810/general-scholarship-1910px.png';
```
Comment (DFurnes) [inline resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js:12]: I love this approach of scaffolding out with hard-coded content/images!

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  <>
+    <SiteNavigationContainer />
+
+    <main>
+      <article>
+        <img
+          data-testid="vr-marketing-page-banner-image"
+          srcSet={srcset}
+          src={contentfulImageUrl(coverImage, '1400', '450', 'fill')}
+          alt="Decorative banner"
```
Comment (DFurnes) [inline resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js:33]: I'm imagining how a screen reader might approach reading this out... would an empty `alt=""` make more sense? This seems to be the [suggested approach](https://www.w3.org/WAI/WCAG21/Techniques/html/H67) from the WCAG, but not sure if our linter rules will agree!

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        <div
+          data-testid="vr-marketing-page-banner-main"
+          style={{ backgroundColor: '#309450' }}
+          className="p-4"
+        >
+          <img
+            data-testid="vr-marketing-page-banner-logo"
+            className="m-auto"
+            src={contentfulImageUrl(logo, '250', '60')}
+            alt="Logo"
```
Comment (DFurnes) [inline resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js:45]: I'm not sure if we'll have this handy in the Contentful type for this page, but same train of thought as my last question – would `${partnerName} logo` be more descriptive for folks using screen readers?

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            className="text-white text-center text-lg"
+          >
+            Take 2 minutes to register to vote at your current address.
+          </h2>
+        </div>
+
+        <SocialShareTray
+          className="text-center"
+          // Pass through the current URL without the query parameters.
+          shareLink={window.location.href.split('?')[0]}
```
Comment (DFurnes) [inline resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js:66]: Would `${window.location.origin}{window.location.pathname}` be safer? It's probably unlikely, but wondering if a URL with an escaped `\?` or some other funny business might cause us trouble here.

**Review by DFurnes — APPROVED** [review]
Looking great! 🗳️

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js (hunk 2 above)
Comment (mendelB) [reply resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js:33]: Interesting! My intention at this point was just to quiet the linter and replace this with the `description`. But hmm, if _that_ is left blank in Contentful I can see if they'll let us get away with an empty `alt=""`?

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js (hunk 3 above)
Comment (mendelB) [reply resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js:45]: ^ That's exactly what I was hoping to replace this with once we introduce the Contentful content! (And ofc optimally a helpful `description` from the Asset, but that's our ideal universe 🙏 )

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js (hunk 4 above)
Comment (mendelB) [reply resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js:66]: That does seem safer! I copied this over from the [Story Page](https://github.com/DoSomething/phoenix-next/blob/45e94c9c339f576f8e696783d76e7ec5d8c9e66c/resources/assets/components/pages/StoryPage/StoryPage.js#L51-L56), so I'll replace it in both.

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js (hunk 2 above)
Comment (DFurnes) [reply resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js]: A ha! Smart! 👍

File: resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js (hunk 3 above)
Comment (DFurnes) [reply resources/assets/components/pages/VoterRegistrationMarketingPage/VoterRegistrationMarketingPage.js]: Of course, prefect! 👍

**Pull request merged**